tests/jenkins/TestDataPrepperDistributionArtifacts.groovy

```diff
@@ -3,35 +3,60 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import jenkins.tests.BaseJobPipelineTest
+
+import jenkins.tests.BuildPipelineTest
 import org.junit.Before
 import org.junit.Test
 
-class TestDataPrepperDistributionArtifacts extends BaseJobPipelineTest {
+class TestDataPrepperDistributionArtifacts extends BuildPipelineTest {
 
     @Before
     void setUp() {
+
+        def filenamesForUrls = ['tar_dummy_1_artifact_1.0.0.tar.gz', 'tar_dummy_1_artifact_1.0.0.tar.gz.sig',
+                                'tar_dummy_2_artifact_1.0.0.tar.gz', 'tar_dummy_2_artifact_1.0.0.tar.gz.sig']
+
+        def artifactsPath = "${this.workspace}/release/archives/linux/build/distributions"
+
+        def bucketName = 'job-s3-bucket-name'
+
+        this.registerLibTester(new TestUploadToS3(
```

Comment:
It isn't as clear that relationship of TestUploadToS3 that it can be a 'libTester', maybe workshop another way to name them to make this relationship cleaner?

Reply:
Updated the class to have `LibTester`